recipes/libdrs_c/build.sh

```diff
@@ -0,0 +1,17 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC $CFLAGS"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+
+if [ `uname` == Linux ]; then
+    MAKEFILE=libdrs_Makefile.Linux.gfortran
+    echo "Linux  "${PREFIX}
+else
+    MAKEFILE=libdrs_Makefile.Mac.gfortran
+    echo "Mac  "${PREFIX}
+fi
+cd lib
+sed "s#@cdat_EXTERNALS@#${PREFIX}#g;" ${MAKEFILE}.in > Makefile
+make  
+make  install
```

Comment:
I just looked ans there is not `make check` or no `make test`, this is a very old library for people who use the `drs` file format.